ci/pipeline.yml

```diff
@@ -96,10 +96,10 @@ jobs:
             dir: src
             path: ci/tasks/deploy.sh
         params:
-          <<: *staging-cf
-          CF_APP_NAME: pages-builder-staging
+          <<: *cf
+          CF_APP_NAME: pages-builder-((env))
```

Comment:
need to make sure the apps follow this naming convention, currently production is just `pages-builder`

Reply:
We should, but we have had a convention of appending the env to app and service names to make it clear when connecting to things via `connect-to-service` or `run-task`. This predated me, i'll leave it up to y'all to decide what makes the most sense,

 We could also just call it `builder` 🤷